starters/next-wordpress-starter/pages/examples/auth-api.jsx

```diff
@@ -0,0 +1,101 @@
+import Head from "next/head";
+import Link from "next/link";
+import { gql } from "@pantheon-systems/wordpress-kit";
+import { client } from "../../lib/WordpressClient";
+import { getFooterMenu } from "../../lib/Menus";
+import Layout from "../../components/layout";
+
+export default function AuthApiExampleTemplate({ menuItems, SSRPosts }) {
+  return (
+    <Layout footerMenu={menuItems}>
+      <Head>
+        <title>API Authorization Example</title>
+        <meta name="description" content="Generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <div className="prose lg:prose-xl mt-10 flex flex-col mx-auto max-h-screen">
+        <h1>API Authorization Example</h1>
+
+        <Link passHref href="/">
+          <span className="w-full underline cursor-pointer">Home &rarr;</span>
+        </Link>
+
+        <div className="mt-12 max-w-lg mx-auto lg:grid-cols-3 lg:max-w-screen-lg">
+          {SSRPosts?.length > 0 ? (
+            <p>
+              🎉 Next.js was able to successfully make an authenticated request
+              to Wordpress! 🎉
```

Comment:
```suggestion
              to WordPress! 🎉
```
WP gets mad when you don't capitalize the "P"